aether-kernel/aether/kernel/api/views.py

```diff
@@ -53,7 +53,23 @@
 )
 
 
-class ProjectViewSet(MtViewSetMixin, viewsets.ModelViewSet):
+class FilteredMixin(object):
+    @action(detail=False, methods=['delete'], url_path='filtered-delete')
+    def filtered_delete(self, request, *args, **kwargs):
+        qs = super(FilteredMixin, self).get_queryset()
+        filter_class = self.filter_class
+        try:
+            filtered_projects = filter_class(self.request.GET, queryset=qs)
```

Comment:
stupid comment, have to tried to do directly `qs.delete()` the `get_queryset()` method should have already filtered the query set, I think so :thinking:

Reply:
Nope, Filter isn't applied on `get_queryset()`

Reply:
That's weird...